packages/button/src/mwc-button-base.ts

```diff
@@ -34,10 +35,34 @@ export class ButtonBase extends LitElement {
 
   @property() label = '';
 
+  @query('#button') buttonElement!: HTMLElementWithRipple;
+
   protected createRenderRoot() {
     return this.attachShadow({mode: 'open', delegatesFocus: true});
   }
 
+  focus() {
+    const buttonElement = this.buttonElement;
+    if (buttonElement) {
+      const ripple = buttonElement.ripple;
+      if (ripple) {
```

Comment:
Add a comment? Why does ripple need to know about focus? I imagine it just as background animation that you wouldn't think has any bearing on what's focused.